app/mac/BUILD.gn

```diff
@@ -9,11 +9,7 @@ chrome_framework_name = chrome_product_full_name + " Framework"
 chrome_helper_name = chrome_product_full_name + " Helper"
 
 # This must be same value in //chrome/BUILD.gn.
-if (new_mac_bundle_structure) {
```

Comment:
what was the default value of this before? I'm not sure that `chrome_version_full` is the correct value

Reply:
Before, it was set to `A`... but was changed with https://github.com/brave/brave-core/commit/f102ceee4916655672ea1014933076d53be861f4#diff-068a0d535a7ec8fe6294c63fca618ecd (new mac signing)

`new_mac_bundle_structure` flag then removed with https://chromium.googlesource.com/chromium/src/+/d24fe6f65d3e2fe6f214f007c7b9d4cc5c347065

Reply:
looks like default was `true` so this should be ok